app/models/api/hses.rb

```diff
@@ -1,25 +1,52 @@
+require "api_request"
 require "fake_api_response_wrapper"
 require "fake_issues"
 
-class Api::Hses
-  class Issue
-    include FakeApiResponseWrapper
-    attr_accessor :id
+module Api::Hses
+end
+
+class Api::Hses::Issue < ApiRequest
+  include FakeApiResponseWrapper
+  attr_accessor :id
+
+  def initialize(id:)
+    @id = id
+  end
+
+  # TODO update this with real data once we have /issue endpoint
+  def request
+    details_wrapper.merge(
+      data: FakeIssues.instance.json[:data].sample
+    )
+  end
+
+  private
+
+  def host
+    Rails.configuration.x.hses.api_base
```

Comment:
can this go in Api::Hses so it doesn't have to be repeated?

Reply:
I think this is our nested classes problem again :(  Maybe I can figure out some way of importing the methods from another class without inheriting them.

Reply:
I added it as a method on the module Api::Hses and included the module directly, which has done the trick, even if not particularly elegantly!